### PR TITLE
Remove the @internal tag

### PR DESCRIPTION
**Pull request by leofeyer**
This is the PR for #390. It generally removes the tag and adds comments where it is kept.

### CODE REVIEW

File: src/Framework/Adapter.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
  *
  * @author Yanick Witschi <https://github.com/toflar>
  *
- * @internal
+ * @internal Do not instantiate this class in your code. Use ContaoFramework::getAdapter() instead.
```
Comment (aschempp) [inline src/Framework/Adapter.php]: :+1:

**Comment by aschempp** [comment]
Looks good to me for a first shot :D

**Comment by Toflar** [comment]
:+1:

**Comment by discordier** [comment]
Same for me.

**Comment by leofeyer** [comment]
Merged in e60e85a253591b27ba6fd444c998bda45fbbc9aa.

**Pull request merged**